tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -242,14 +243,19 @@ void onRenderTaskCancelled() {
 
   void onRenderTaskPostExecute() {
     mIsRendering = false;
-    if( !mTransitionsEnabled ) {
-      cleanup();
-    }
-    if( mTileRenderListener != null ) {
-      mTileRenderListener.onRenderComplete();
-    }
-    invalidate();
-    requestRender();
+    mTileRenderHandler.post( new Runnable() {
```

Comment:
One of the benefits of using B is that we can cancel it during execution (if required, but I can't see any case that this could be used). By performance I don't know which one is more efficient. 

Now considering the `addTileToCurrentTileCanvasView` case. I believe that we need to create new instances of runnable exactly because of the Tile passed as parameter (this method is called by every tile not only once per rendering like the `onRenderTaskPostExecute`